test/feature/windows/agent_config.json

```diff
@@ -28,6 +128,24 @@
             ],
             "metrics_collection_interval": 1
         },
+        "TCPv4": {
```

Comment:
Is this the name of a performance object on all the supported Windows operating systems?

Reply:
Yes it is. It was tested on all supported Windows OS and they successfully validated the metric.